.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
-image: $CI_REGISTRY/swe-database-team/gitlab-ci/whois-build:21fc0efa
+image: $CI_REGISTRY/swe-database-team/gitlab-ci/whois-build:9704969e
```

Comment:
Temporal commit: we have to do the change once we have the final commit

Reply:
Updated to v0.0.5 - java21, amazoncorretto, node18, firefox114